### PR TITLE
Revert 4358

### PR DESCRIPTION
**Pull request by greg0ire**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Removed
- recently introduced checkbox-disabling feature, which was not stable enough

```

## Subject

This reverts #4358 (partially)

### CODE REVIEW

File: Resources/public/Admin.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ var Admin = {
             padding: 15,
             overflow: 'auto'
         });
-
+
```
Comment (jordisala1991) [inline Resources/public/Admin.js]: Can you try to remove this whitespaces?

**Comment by jordisala1991** [comment]
Thanks @greg0ire , and @RomainSanchez we can try another approach to the problem.

File: Resources/public/Admin.js (hunk 1 above)
Comment (OskarStark) [inline Resources/public/Admin.js]: Lets keep it for now and create a release

**Comment by OskarStark** [comment]
Thanks @greg0ire 👍

**Pull request merged**